### PR TITLE
Add 'TeX Live' section.

### PR DESCRIPTION
**Pull request by flexibeast**
Today i got around to installing TeX Live, and found the wiki page about it helpful. With the wiki going away, i think it would be useful to add this information to the handbook.

### CODE REVIEW

**Comment by Duncaen** [comment]
For commands that need to be run as root, we use `#` as prefix instead of `$` without sudo.
https://docs.voidlinux.org/contributing/void-docs/style-guide.html#commands

I think instructions like `xbps-install textlive-bin` are not necessary, users should already know how to install packages and/or refer to the right documentation which describes how to install packages.
I would also add the `gnupg` requirement to the initial block where it says to install `texlive-bin` and maybe also mention that `texlive-bin` is the latest version and other versions like `texlive2018-bin` are available.

**Comment by flexibeast** [comment]
Thanks for the quick feedback!

Sorry for not having changed the `$ sudo ...` to `# ...` - yes, i'd read about `$` vs `#` in the style guide, but wasn't sure if there was some particular reason the wiki had specified the use of `sudo` instead.

i've now updated the PR with the requested changes. However, unfortunately i forgot to do a `git --amend` before force-pushing, so the PR now has two commits - my apologies. i hope that doesn't cause too much inconvenience.

**Comment by leahneukirchen** [comment]
Looks good.  You could also mention `tlmgr install scheme-full` to install all available packages.

**Comment by flexibeast** [comment]
Thanks for the suggestion; i've updated the PR accordingly.

**Comment by flexibeast** [comment]
Ping?

File: src/config/texlive.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+$ tlmgr search --global <package-name>
+```
+
+and then install it:
+
+```
+# tlmgr install <package_name>
+```
+
+If the error is a missing file and not a package (for example a font), search
```
Comment (jnbr) [inline src/config/texlive.md]: Nitpick: "the error" appears the first time here.

**Comment by flexibeast** [comment]
@jnbr: Thanks for the feedback; i've now fixed that section. Hope all is now in order.

**Comment by jnbr** [comment]
Thank you. Looks good to me now.

**Comment by bobertlo** [comment]
I don't know about TeX Live, but there seems to be a consensus and I don't see any glaring issues otherwise.

**Pull request merged**